metagpt/actions/write_code.py

```diff
@@ -135,20 +156,44 @@ async def run(self, *args, **kwargs) -> CodingContext:
         return coding_context
 
     @staticmethod
-    async def get_codes(task_doc, exclude) -> str:
+    async def get_codes(task_doc, exclude, mode="normal") -> str:
```

Comment:
Please add comments to the function, specifically for the 'mode' parameter, to clarify its purpose.